### PR TITLE
replace bcolors with termcolor

### PR DESCRIPTION
**Pull request by iMartzen**
## Issue number and link

Closes: https://github.com/Hacker0x01/hai-on-hackerone/issues/6

### 📝 Summary

This merge request is linked to issue #6. The main change is updating the print statement for colored console output. The previous method using `bcolors` has been replaced with the `colored` function for better readability and maintainability.

### 🧪 Test Plan

1. Run the script to verify that the output "Response from Hai" is displayed in light blue.
2. Ensure that there are no errors related to the `colored` function.
3. Check the terminal to confirm the color change from the old method to the new method.

### 🤔 Additional remarks

Switching to the `colored` function simplifies the code and improves consistency in how colors are applied across the project.

### ✅ Review Checklist

* [ ] Linked to the relevant Github issue
* [ ] MR status changed from draft to ready for review
* [ ] Comprehensive summary of the changes is written
* [ ] I ensured that unit, integration, regression, and end-to-end tests are added (if applicable)
* [ ] I assessed the performance impact of the changes (if applicable)
* [ ] I conducted a thorough self-review of the code on Github
* [ ] I ensured that all documentation is updated to reflect the changes

### CODE REVIEW

File: cli/hai.py [hunk 1]
```diff
@@ -76,8 +77,8 @@ async def send_to_hai(report, verbose):
     if verbose:
         end_time = time.time()
         execution_time = end_time - start_time
-        print(f"{bcolors.OKCYAN}Execution Time: {execution_time} seconds{bcolors.ENDC}")
-        print(f"{bcolors.OKBLUE}Responses from Hai: {bcolors.ENDC}")
+        print(colored(f"Execution Time: {execution_time} seconds", 'cyan'))
```
Comment (martijnrusschen) [inline cli/hai.py:80]:
```suggestion
        print(colored("Execution Time: {execution_time} seconds", 'cyan'))
```

Comment (martijnrusschen) [reply cli/hai.py:80]: I suppose this was a mistake?

File: cli/hai.py [hunk 2]
```diff
@@ -122,18 +123,18 @@ async def send_individual_prompt(prompt, report, verbose):
...
             async with session.post('https://api.hackerone.com/v1/hai/chat/completions', auth=aiohttp.BasicAuth(settings.api_name, settings.api_key), json=data, headers=settings.headers) as r:
                 response_data = await r.json()
                 if verbose:
-                    print(f"{bcolors.OKBLUE}Response from Hai{bcolors.ENDC}")
+                    print(colored("Response from Hai", 'blue'))
                     print(response_data)
                 return await wait_for_hai(response_data, verbose)
         except Exception as err:
-            print(f"{bcolors.FAIL}Unexpected {err=}, {type(err)=}{bcolors.ENDC}")
+            print(colored(f"Unexpected {err=}, {type(err)=}", 'light_red'))
```
Comment (martijnrusschen) [inline cli/hai.py:137]: Same?

File: cli/reports.py [hunk 3]
```diff
@@ -141,11 +141,12 @@ async def show_reports(response, verbose, comment_hai_flag, custom_field_hai_fla
...
-        print(f"{bcolors.OKCYAN}Sending report {report} to Hai...{bcolors.ENDC}")
+        # print(colored(f"{banner}", 'light_magenta'))
+        print(colored(f"Processing report {counter} of {len(report_ids)}", 'cyan'))
+        print(colored(f"Sending report {report} to Hai...", 'cyan'))
         predictedValidity, predictedValidityCertaintyScore, predictedValidityReasoning, predictedComplexity, predictedComplexityCertaintyScore, predictedComplexityReasoning, predictedOwnershipCertaintyScore, predictedOwnershipReasoning, productArea, squadOwner = await send_to_hai(report, verbose)
         hai_actions(predictedValidity, predictedValidityCertaintyScore, predictedValidityReasoning, predictedComplexity, predictedComplexityCertaintyScore, predictedComplexityReasoning, predictedOwnershipCertaintyScore, predictedOwnershipReasoning, productArea, squadOwner, report, comment_hai_flag, custom_field_hai_flag, csv_output_flag, verbose)
-    print(f"{bcolors.OKCYAN}{len(report_ids)} reports has been successfully processed{bcolors.ENDC}")
+    print(colored(f"{len(report_ids)} reports have been successfully processed", 'cyan'))
```
Comment (martijnrusschen) [inline cli/reports.py:149]: Same here

**Review by martijnrusschen — APPROVED** [review]

File: cli/hai.py (hunk 1 above)
Comment (iMartzen) [reply cli/hai.py:80]: No they are f-strings: https://docs.python.org/3/reference/lexical_analysis.html#f-strings

**Pull request merged**